pkg/txm/clientwrappers/dualbroadcast/meta_client.go

```diff
@@ -0,0 +1,614 @@
+      {
+        "name": "userOp",
+        "type": "tuple",
+        "components": [
+          { "name": "from", "type": "address" },
+          { "name": "to", "type": "address" },
+          { "name": "value", "type": "uint256" },
+          { "name": "gas", "type": "uint256" },
+          { "name": "maxFeePerGas", "type": "uint256" },
+          { "name": "nonce", "type": "uint256" },
+          { "name": "deadline", "type": "uint256" },
+          { "name": "dapp", "type": "address" },
+          { "name": "control", "type": "address" },
+          { "name": "callConfig", "type": "uint32" },
+          { "name": "dappGasLimit", "type": "uint32" },
+          { "name": "solverGasLimit", "type": "uint32" },
+          { "name": "bundlerSurchargeRate", "type": "uint24" },
+          { "name": "sessionKey", "type": "address" },
+          { "name": "data", "type": "bytes" },
+          { "name": "signature", "type": "bytes" }
+        ]
+      },
+      {
+        "name": "solverOps",
+        "type": "tuple[]",
+        "components": [
+          { "name": "from", "type": "address" },
+          { "name": "to", "type": "address" },
+          { "name": "value", "type": "uint256" },
+          { "name": "gas", "type": "uint256" },
+          { "name": "maxFeePerGas", "type": "uint256" },
+          { "name": "deadline", "type": "uint256" },
+          { "name": "solver", "type": "address" },
+          { "name": "control", "type": "address" },
+          { "name": "userOpHash", "type": "bytes32" },
+          { "name": "bidToken", "type": "address" },
+          { "name": "bidAmount", "type": "uint256" },
+          { "name": "data", "type": "bytes" },
+          { "name": "signature", "type": "bytes" }
+        ]
+      },
+      {
+        "name": "dAppOp",
+        "type": "tuple",
+        "components": [
+          { "name": "from", "type": "address" },
+          { "name": "to", "type": "address" },
+          { "name": "nonce", "type": "uint256" },
+          { "name": "deadline", "type": "uint256" },
+          { "name": "control", "type": "address" },
+          { "name": "bundler", "type": "address" },
+          { "name": "userOpHash", "type": "bytes32" },
+          { "name": "callChainHash", "type": "bytes32" },
+          { "name": "signature", "type": "bytes" }
+        ]
+      },
+      { "name": "gasRefundBeneficiary", "type": "address" }
+    ],
+    "outputs": [ { "name": "auctionWon", "type": "bool" } ]
+  }
+]`
+	ABI = `[
+  {
+    "inputs": [
+      {
+        "internalType": "address",
+        "type": "address"
+      },
+      {
+        "internalType": "bytes",
+        "type": "bytes"
+      }
+    ],
+    "name": "update",
+    "type": "function"
+  }
+]`
+)
+
+type MetaClientKeystore interface {
+	SignMessage(ctx context.Context, address common.Address, data []byte) ([]byte, error)
+	SignTx(ctx context.Context, fromAddress common.Address, tx *evmtypes.Transaction) (*evmtypes.Transaction, error)
+}
+
```

Comment:
Nice to add type assertion:
```suggestion
var _ txm.Client = (*MetaClient)(nil)
```